master/buildbot/test/unit/db/test_logs.py

```diff
@@ -136,14 +136,16 @@ def checkTestLogLines(self):
             'another line',
             'yet another line',
         ]
+
+        def _join_lines(lines: list[str]):
```

Comment:
I think this should be a separate commit. I got confused that the behavior is being changed, where in fact it is not.